root/doc/EditTypeIndex.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import Layout from '../layout/index.js';
+
+type EditTypePropertyProps = {
+  +className?: 'even',
+  +label: string,
+  +property: string,
+};
+
+type EditTypeIndexProps = {
+  +editType: {
+    +appliesAutomatically: boolean,
+    +canBeApproved: boolean,
+    +editCategory: string,
+    +editName: string,
+  },
+  +page?: DocPageT,
+};
+
+function checkboxOrCross(condition: boolean) {
+  return condition ? l('✔') : l('✕');
+}
+
+const EditTypeProperty = ({
+  className,
+  label,
+  property,
+}: EditTypePropertyProps): React$Element<'tr'> => (
+  <tr className={className}>
+    <th>{label}</th>
+    <td>{property}</td>
+  </tr>
+);
+
+const EditTypeIndex = ({
+  editType,
+  page,
+}: EditTypeIndexProps): React$Element<typeof Layout> => (
+  <Layout fullWidth noIcons title={editType.editName}>
+    <h1>{editType.editName}</h1>
+
+    <p>
+      <a href="/doc/Edit_Types">{lp('Edit types', 'noun')}</a>
+      {' > '}
+      {editType.editCategory}
+      {' > '}
+      {editType.editName}
+    </p>
+
+    {page ? (
+      <>
+        <h2>{l('Description')}</h2>
+        <div dangerouslySetInnerHTML={{__html: page.content}} />
+      </>
+    ) : null}
+
+
+    <h2>{l('Details')}</h2>
+    <table className="tbl edit-type">
+      <tbody>
+        <EditTypeProperty
+          label={l('Applies automatically')}
+          property={checkboxOrCross(editType.appliesAutomatically)}
+        />
+        <EditTypeProperty
+          className="even"
+          label={l('Can be approved')}
+          property={checkboxOrCross(editType.canBeApproved)}
+        />
```

Comment:
"Applies automatically" is a useful addition. 👍

I feel like an "x" there can be misleading, since it may possibly still be applied automatically by auto-editors. Which is what "Can be approved" indicates, but in a weird, indirect way? (And a checkmark can be similarly misleading, since it may still be made votable by choice or by editor restrictions.) I guess it would be clearer to me if this were a single property...

"Can be applied automatically: Never/By normal editors/By auto-editors only".

("By auto-editors only" would imply it can be approved by them.)

Am I missing some case where these make sense separated?

Reply:
Probably not, so I made the change. This made me feel we need a second row "Required privileges" because "Can be applied automatically: By any editor" for `Add area` is kind of funny, but AFAICT we do not store the required privileges on the edit class itself... maybe we should?